packages/web-client/app/helpers/get-previous-visible-postable.ts

```diff
@@ -0,0 +1,25 @@
+import { helper } from '@ember/component/helper';
+import { WorkflowPostable } from '../models/workflow/workflow-postable';
+
+function getPreviousVisiblePostable([postable]: [WorkflowPostable]) {
+  if (!postable) {
+    return null;
+  }
+
+  let workflowVisiblePostables = postable!.workflow!.peekAtVisiblePostables();
+  let result = null;
+
+  for (let candidate of workflowVisiblePostables) {
+    if (candidate === postable) {
+      return result;
+    }
+
+    if (!(candidate as any)._isCheck) {
```

Comment:
Yes, because it doesn't render anything. It's not ideal, but couldn't find a good way to add on a check in a way that is in sync with a component but doesn't end up racing with async processes.

`_isCheck` was something I did earlier when I wasn't sure about the structure of the model. I should probably change it to `isCheck`.

Reply:
@backspace is onto something here.